rubocop.gemspec

```diff
@@ -42,7 +42,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency('ruby-progressbar', '~> 1.7')
   s.add_runtime_dependency('unicode-display_width', '~> 1.4.0')
 
-  s.add_development_dependency('bundler', '~> 1.3')
+  s.add_development_dependency('bundler', '>= 1.3.0')
```

Comment:
It seems preferable to specify the upper limit of dependent gem.

```diff
-s.add_development_dependency('bundler', '>= 1.3.0')
+s.add_development_dependency('bundler', '>= 1.3.0', '< 3.0')
```

cf. https://github.com/ruby/ruby/commit/cfa6c55347d037b152a4b93a390c73dfee6dccbd

Reply:
OK. I fixed.